seaworthy/checks.py

```diff
@@ -17,6 +17,18 @@ def docker_client():
     client.api.close()
 
 
+def fetch_images(images):
+    """
+    Fetch images if they aren't already present.
+    """
+    with docker_client() as client:
+        for image in images:
+            try:
+                client.images.get(image)
+            except docker.errors.ImageNotFound:  # pragma: no cover
+                client.images.pull(image)
```

Comment:
This is replicating logic in docker_helper.. not sure if we can do something about that? Also, the pull could take a really long time but there's no log output.

Reply:
Yeah. I should probably log something, but I didn't want to spend the time setting that up.

Reply:
This bit is my only outstanding issue with this PR. I would like this logic to be in one place, ideally somewhere in the `dockerhelper` module, I think.

Reply:
Also I already want this kind of thing for my pytest fixture tests :)